src/reducers/tabs.js

```diff
@@ -184,9 +194,13 @@ export const getSourceTabs = createSelector(
   getSources,
   getUrls,
   (tabs, sources, urls) =>
-    tabs.filter(tab =>
-      getSourceByUrlInSources(sources, urls, tab.url, tab.isOriginal)
-    )
+    tabs.filter(tab => {
```

Comment:
The `tab => { }` function looks the same so maybe create a function to reuse the logic in both spots?

```js
.filter(getTabWithOrWithoutUrl(tab, sources, url))
```

Something like that.